(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'DAL' object has no attribute 'posts'"
p3
sS'layer'
p4
S'/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/api.py'
p5
sS'code'
p6
S'<code object <module> at 0x7feae8158b30, file "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/api.py", line 2>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/api.py", line 24, in <module>\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/gluon/globals.py", line 421, in <lambda>\n    self._caller = lambda f: f()\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/api.py", line 10, in get_all_post\n    posts = db(db.posts).select()   #get all posts entries in post table\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/gluon/packages/dal/pydal/base.py", line 673, in __getattr__\n    return BasicStorage.__getattribute__(self, key)\nAttributeError: \'DAL\' object has no attribute \'posts\'\n'
p11
s.